test/scaladoc/run/t8755b/ploogin_1.scala

```diff
@@ -0,0 +1,29 @@
+
+package t8755
+
+import scala.tools.nsc.{Global, Phase}
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+import scala.reflect.io.Path
+import scala.reflect.io.File
+
+/** A test plugin.  */
+class Ploogin(val global: Global) extends Plugin {
+  import global._
+
+  val name = "ploogin"
+  val description = "A sample plugin for testing."
+  val components = List[PluginComponent](TestComponent)
+
+  private object TestComponent extends PluginComponent {
+    val global: Ploogin.this.global.type = Ploogin.this.global
+    override val runsBefore = List("jvm")
+    val runsAfter = List("typer", "refchecks")
```

Comment:
I see, https://github.com/scala/scala/pull/10856 drops the phase because one of the `runsAfter` constraints is invalid. I think either way is fine here.

2.13.14 scaladoc drops the "refchecks" constraint and runs the phase after "typer".

Reply:
Yes I should have documented it: if runsAfter means "depends on", then all those phases are required, and not "at least one".